packages/unlock-js/src/PublicLock/v10/extendKey.js

```diff
@@ -18,6 +18,7 @@ export default async function (
   {
     lockAddress,
     tokenId,
+    owner,
```

Comment:
I am prettu sure `owner` is not going to be consistently defined. a MUCH better approach would be to retrieve the owner of the key.